endpoint_logger_response.py

```
@@ -0,0 +1,81 @@
+
+        log_entry = {
+            "event": "request",
+            "method": request.method,
+            "url": request.url,
+            "path": request.path,
+            "host": request.host,
+            "port": request.port,
+            "query": dict(request.query),
+            "headers": dict(request.headers),
+            "body": body_text,
+            "timestamp": int(time.time())
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)